fetch.sh

```diff
@@ -33,7 +33,7 @@ do
     echo "$i"
     deb="deb/$(basename $i)"
     if [ ! -f "$deb" ]; then
-        curl -Ls "$i" > "$deb"
+        wget -O "$deb" "$i"
```

Comment:
I'd like to keep using curl here. `wget` is not available by default on most Linux distro's (I don't know why Ubuntu ships wget and not curl), and same for macOS (where I run this script sometimes)

In general I trust curl more than wget to be reliable, too.